reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+---
+title: Migrating from Windows PowerShell 5.1 to PowerShell 7
+description: Migrate to PowerShell 7 with these helpful notes
+ms.date: 03/25/2020
+---
+
+# Migrating from Windows PowerShell 5.1 to PowerShell 7
+
+PowerShell 7 is a cross-platform (Windows, Linux, and macOS) automation tool and configuration
+framework optimized for dealing with structured data, REST APIs, and object models. PowerShell
+includes a command-line shell, object-oriented scripting language, and a set of tools for executing
+scripts/cmdlets and managing modules. Designed for Cloud, Hybrid-Cloud and on-premises, the latest
+release of PowerShell is packed with performance enhancements and features over previous versions.
```

Comment:
I think we should highlight the side-by-side nature of 5.1 and 7 early on and how safe it is to deploy over time